java/src/main/native/CMakeLists.txt

```diff
@@ -146,11 +151,13 @@ endif (CUDF_INCLUDE AND CUDF_LIBRARY)
 
 find_path(RMM_INCLUDE "rmm"
           HINTS "$ENV{RMM_ROOT}/include"
+                "$ENV{RMM_HOME}/include"
```

Comment:
What's going on with `RMM_HOME`?  I don't see this being used in the cpp CMakeLists.txt, and given there's already an `RMM_ROOT` used by both this and the cpp CMakeLists.txt, I would not expect there to be a need for `RMM_HOME` especially since we don't see it in the cpp build.

Reply:
cpp CMakeLists.txt doesn't have `RMM_HOME` as a hint and the reason its working when we build it in rapids-compose is because we can directly pass the cmake variables but unfortunately there is no way that I and @trxcllnt could find to override those variables conditionally in `antrun` plugin in maven. so when we don't pass any variables to maven they are passed as empty strings which causes the cmake to assign those values to the `RMM_INCLUDE` etc causing build failures. 

One thing that I completely forgot to mention in our chat offline, and as Paul has pointed out in his comment, is that in rapids-compose we are doing source building so, there is no install. cudf and rmm both don't copy their headers to the build directory and neither have a lib folder in the build directory. 

I hope this justifies this change